salt/modules/vmctl.py

```diff
@@ -257,6 +249,14 @@ def start(
     if disks and len(disks) > 0:
         cmd.extend(["-d", x] for x in disks)
 
+    if not (name or id):
+        raise SaltInvocationError('Must provide either "name" or "id"')
```

Comment:
I'd suggest keeping this particular check at the original location. This way we can error out early if the `name` or `id` are missing.

The `elif name [..]` can stay where you moved them to.

Reply:
Thanks for the feedback, just added another commit (#d080992) to fix that.

Reply:
That looks fine to me. Could you please add a changelog entry too?

Reply:
Yep, that's done :)